File: tests/class/computed-getter.js
export function isEmpty$size$(size) {
  return size.isEmpty;
}
export function isEmptyInlined$size$(size) {
  return size.width === 0 && size.height === 0;
}
export class Size {
  get isEmpty() {
    return this.width === 0 && this.height === 0;
  }

}

File: tests/class/computed-getter.swift
public class Size {
	var width: Double
	var height: Double

	init(width: Double, height: Double) {
	}

	public var isEmpty: Bool {
		return width == 0 && height == 0
	}

	var isEmptyInlined: Bool {
		return width == 0 && height == 0
	}
}

public func isEmpty(size: Size) -> Bool {
	return size.isEmpty
}

public func isEmptyInlined(size: Size) -> Bool {
	return size.isEmptyInlined
}
